website/pages/explore/dir-content/stack-devonfw-ide.asciidoc

```diff
@@ -1,14 +0,0 @@
-= devonfw ide FILE
-
-[.directory]
-== devonfw ide
-
-The devon-ide is a fantastic tool to automatically download, install, setup and update the IDE (integrated development environment) of your software development projects.
-
-
-[.common-links]
-=== Links
-
-<</website/pages/docs/getting-started-the-devon-ide.asciidoc.html#, devonfw ide docs>>
-
-https://github.com/devonfw/ide/wiki[see in github]
```

Comment:
please restore it, you could add another section toplevel "Accelerators" and put IDE and CobiGen inside